command/whitelist/deployment/result.go

```diff
@@ -0,0 +1,32 @@
+package deployment
+
+import (
+	"bytes"
+	"fmt"
+
+	"github.com/0xPolygon/polygon-edge/types"
+)
+
+type DeploymentResult struct {
+	AddAddress    []types.Address `json:"addAddress"`
+	RemoveAddress []types.Address `json:"removeAddress"`
```

Comment:
Should you omit these if they're empty?

Reply:
Fixed d2f59d62cb14506a27767dd3735443f9b42bb8b1